sdk/eventhub/Azure.Messaging.EventHubs/assets/live-tests-azure-setup.ps1

```diff
@@ -158,15 +158,15 @@ try
     Write-Host "Done."
     Write-Host ""
     Write-Host ""
-    Write-Host "EVENT_HUBS_RESOURCEGROUP=$($resourceGroupName)"
+    Write-Host "EVENTHUB_RESOURCE_GROUP=$($resourceGroupName)"

```

Comment:
These should remain "EVENT_HUBS"; it's the name of the product.

Reply:
I would be happy to do it but the test infrastructure uses the directory name under sdk as the prefix and eventhubs uses eventhub directory. Not sure what to do here.

Reply:
Well.... yeah, that's a bummer.  The correct answer here is to finally get permission to fix the repository path, it would seem.   I suppose it's either that or working some kind of alias into the test infrastructure, but that hardly seems worth the effort.   I'll follow-up with revisiting the topic with the engineering team.